src/xmipp/SConscript

```diff
@@ -225,20 +247,37 @@ PROGRAMS_WITH_PYTHON = ['volume_align_prog','mpi_classify_CLTomo_prog']
 PROGRAMS_WITH_OPENCV = ['movie_optical_alignment_cpu','mpi_volume_homogenizer']
 PROGRAMS_WITH_OPENCV_AND_CUDA = ['movie_optical_alignment_gpu']
 for p in glob(os.path.join(XMIPP_PATH,'applications','programs','*')):
-	pname = os.path.basename(p)
```

Comment:
This part was indented with tabs, while the whole rest of the file was indented with spaces. I have changed it for consistency.